genotype/store/trending.py

```diff
@@ -0,0 +1,62 @@
+"""Bla bla. What do you want here???"""
+from genotype.store.models import Analysis, Genotype, Sample
+
+
+def build_snp_dict(analysis_id):
+    """Building a dict of snps for a specific analysis."""
+
+    snp_dict = {}
+    try:
+        genotypes = Genotype.query.filter(Genotype.analysis_id == analysis_id).all()
+    except:
+        genotypes = []
+    for genotype in genotypes:
+        snp_dict[genotype.rsnumber] = [genotype.allele_1, genotype.allele_2]
+
+    return snp_dict
+
+
+def compare(analysis_1, analysis_2):
+    """Compare inernal and external snps"""
+
+    compare_dict = {}
+
+    for snp in set(list(analysis_1.keys()) + list(analysis_2.keys())):
+        if analysis_1.get(snp) == analysis_2.get(snp):
+            compare_dict[snp] = True
+        else:
+            compare_dict[snp] = False
+
+    return compare_dict
+
+
+def prepare_trending(sample_id=None, sample=None):
```

Comment:
The "presentation" should be made in the app that needs the data in a specific manner

Reply:
Again.. Not sure what you mean by presentation. The doc string?